src/components/SkeletonLoading.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+	<div class="placeholder-main" :class="'placeholder-main-' + type">
+		<!-- Placeholder animation -->
+		<template v-for="(suffix, gradientIndex) in ['-regular', '-reverse']">
+			<svg :key="'gradient' + suffix" :class="'placeholder-gradient placeholder-gradient' + suffix">
```

Comment:
This whole `placeholder-gradient` `<svg>` object seems useless to me. Maybe a leftover` it has `position: fixed` and I don't see where it actually results in something being displayed. So maybe it can be removed altogether?

Reply:
It is the gradient animation which is needed.

Reply:
Makes sense, thanks for pointing it out.